packages/eslint-plugin-foreman/bin/tfm-lint.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const path = require('path');
+
+const cwd = process.cwd();
+const chalk = require('chalk');
+const rules = require('../lib/index.js');
+const { CLIEngine } = require('eslint');
+
+let cli = null;
+
+initCLI();
+const results = lintFiles(['/webpack', '/script']);
+formatting(results);
+
+function initCLI() {
+  cli = new CLIEngine({
+    useEslintrc: false,
+    baseConfig: rules,
+    resolvePluginsRelativeTo: path.resolve(__dirname),
+  });
+}
+
+function lintFiles(files) {
+  const filesList = files.map(file => `${cwd}/${file}`);
+  return cli.executeOnFiles(filesList);
+}
+
+function formatting(report) {
+  if (report.errorCount > 0) {
```

Comment:
In case of an error, shouldn't we exit the script with an error code?

Reply:
these are linting errors, no runtime error

Reply:
Once we have a linting error, the script should return a nonzero exit status so the CI will fail